build/build-image/cross/Dockerfile

```diff
@@ -72,7 +72,7 @@ RUN mkdir $TMPDIR \
             github.com/jteeuwen/go-bindata/go-bindata
 
 # Download and symlink etcd. We need this for our integration tests.
-RUN export ETCD_VERSION=v2.2.1; \
+RUN export ETCD_VERSION=v3.0.10; \
```

Comment:
I think we don't want to change build images and default configs yet.

Can you please just bump godeps in this PR?

@lavalamp - thoughts?